examples/vanilla-extract-react/package.json

```
@@ -7,6 +7,7 @@
     "preview": "vite preview"
   },
   "dependencies": {
+    "@hover-design/react": "workspace:0.1.5",
```

```suggestion
    "@hover-design/react": "*",
```

* helps taking the local version instead of cache